### PR TITLE
Fix YAML indentation for network policy optimization

### PR DESCRIPTION
**Pull request by jab-rh**
Related to OSDOCS-4696.

Looks like the indentation isn't entirely correct. This seems to be for OVN-Kubernetes as well, not OpenShift SDN.

Refs https://issues.redhat.com/browse/OSDOCS-4696.

Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@anuragthehatter, this fixes some indentation in the YAML that I noticed; Does this look okay?

Thanks!

**Comment by huiran0826** [comment]
lgtm, but better for @asood-rh to double check?

File: modules/nw-networkpolicy-optimize-ovn.adoc [hunk 1]
```diff
@@ -3,11 +3,11 @@
 // * networking/network_policy/about-network-policy.adoc
 
 [id="nw-networkpolicy-optimize-ovn_{context}"]
-= Optimizations for network policy with OpenShift OVN
+= Optimizations for network policy with OVN-Kubernetes network plugin
 
 When designing your network policy, refer to the following guidelines:
 
-* For network policies with the same `spec.podSelector` spec,  it is more efficient to use one network policy with multiple `ingress` or `egress` rules, than multiple network policies with subsets of `ingress` or `egress` rules.
+* For network policies with the same `spec.podSelector` spec, it is more efficient to use one network policy with multiple `ingress` or `egress` rules, than multiple network policies with subsets of `ingress` or `egress` rules.
```
Comment (asood-rh) [inline modules/nw-networkpolicy-optimize-ovn.adoc:10]: @jab-rh if in this example 1, the point being put is multiple policies versus single policy, then make test-network-policy as test-network-policy1 and  test-network-policy2.
policy 1
<pre>
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: test-network-policy1
spec:
  podSelector: {}
  ingress:
  - from:
    - podSelector:
        matchLabels:
          role: frontend
</pre>
policy 2
<pre>
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: test-network-policy2
spec:
  podSelector: {}
  ingress:
  - from:
    - podSelector:
        matchLabels:
          role: backend
</pre>

**Comment by asood-rh** [comment]
@jab-rh The overall message put across from the PR as I interpret it is using one rule versus multiple rules or multiple policies to selected pods (source and destination) to which policies are applied.

File: modules/nw-networkpolicy-optimize-ovn.adoc (hunk 1 above)
Comment (jab-rh) [reply modules/nw-networkpolicy-optimize-ovn.adoc:10]: @asood-rh, I think the intent here is to show that one policy with multiple `ingress.from` items is worse than consolidating them together with `matchExpressions` if it is possible to do so.

**Comment by lahinson** [comment]
Link to doc preview: https://63236--docspreview.netlify.app/openshift-dedicated/latest/networking/network_policy/about-network-policy#nw-networkpolicy-optimize-ovn_about-network-policy

**Review by lahinson — COMMENTED** [review]
lgtm

**Pull request merged**